echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -105,13 +105,11 @@ def _reorganize_beam_groups(ed_obj):
 
     # Map Beam --> Sonar/Beam_group1
     if "Beam" in ed_obj.group_paths:
-        ed_obj._tree["Sonar"].add_child(ed_obj._tree["Beam"])
-        ed_obj._tree["Sonar/Beam"].name = "Beam_group1"
+        ed_obj._tree["Sonar/Beam_group1"] = ed_obj._tree["Beam"]
```

Comment:
@lsetiawan could you explain to me why this change needed to occur? It looks like this change is the reason you modify other portions of the conversion code.